v-next/hardhat-node-test-reporter/eslint.config.cjs

```diff
@@ -1,8 +1,11 @@
-const { createConfig } = require("../../config-v-next/eslint.cjs");
+const { createConfig } = require("../../config-v-next/eslint.config.cjs");
 
-module.exports = createConfig(__filename);
+const configs = createConfig(__filename);
 
-module.exports.overrides.push({
+/**
+ * * @type {import("eslint").Linter.Config}
+ */
+const overrideConfig = {
   files: ["integration-tests/**/*.ts"],
```

Comment:
I’m thinking we should add this path to the base test config so it can be shared across all packages, without needing to add this override in each one.